packages/noodl-editor/src/editor/src/views/nodegrapheditor.ts

```diff
@@ -1095,6 +1094,25 @@ export class NodeGraphEditor extends View {
     }
   }
 
+  duplicate() {
+    if (this.readOnly) {
+      return false;
+    }
+
+    const nodeset = this.copySelected();
+    if (nodeset === undefined) return;
+
+    const ns = this.getNodeSetFromClipboard() || this.clipboard;
+    if (!ns) return;
+
+    this.insertNodeSet({
+      nodeset: ns,
+      x: this.latestMousePos.x,
+      y: this.latestMousePos.y,
+      toastMessage: 'Duplicate'
```

Comment:
This should be "Duplicated" to match the past tense of the other action UI copy.

Reply:
Yes, updated!